tests/integration/evm_integration_test.go

```diff
@@ -21,7 +25,7 @@ import (
 // returns the current testing context
 type EVMTestSuite struct {
 	suite.Suite
-	network     network.Network
+	network     *network.UnitTestNetwork
```

Comment:
These are integration tests. The name of the file is `evm_integration_tests`.  So in Integration Tests we should not use `UnitTestNetwork`. This should only be used for Unit tests where accessing the App is valid. The integration test util were designed so the App is not accesible and a more "pure" integration is followed.